src/confluent_kafka/experimental/aio/producer/_kafka_batch_executor.py

```diff
@@ -88,6 +90,23 @@ def _produce_batch_and_poll():
         loop = asyncio.get_running_loop()
         return await loop.run_in_executor(self._executor, _produce_batch_and_poll)
 
+    async def flush_librdkafka_queue(self, timeout=-1):
+        """Flush the librdkafka queue and wait for all messages to be delivered
+
+        This method blocks until all outstanding produce requests are completed
```

Comment:
Nit: should read that it "awaits until all outstanding..." since it's immediately returning the await callable instead of actually blocking

Reply:
Should also mention in the text unless the timeout is set to 0